cmp/cmpopts/example_test.go

```diff
@@ -0,0 +1,125 @@
+package cmpopts_test
+
+import (
+	"fmt"
+	"net"
+	"time"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/google/go-cmp/cmp/internal/flags"
+)
+
+func init() {
+	flags.Deterministic = true
+}
+
+// Use IgnoreFields to ignore fields on a type when comparing.
+// Provide an interface of the type, and the field names to ignore.
+func ExampleIgnoreFields_testing() {
+	// Let got be the hypothetical value obtained from some logic under test
+	// and want be the expected golden data.
+	got, want := MakeGatewayInfo()
+
+	// Note that the Diff will still potentially show ignored fields as different,
```

Comment:
> will still potentially show ignored fields as different,

Is still true? I recall fixing the reporter to stop printing ignored fields and using `...` instead, even for adjacent fields.

Reply:
https://github.com/google/go-cmp/blob/d713870ac17fdb9ee5e2ee48ff6562dfb1c0157b/cmp/report_compare.go#L258-L265

Reply:
The example itself demonstrates what I was trying to say.  I ignore `IPAddress` but it still shows up in the diff.  I assume that the reason I added 'potentially' to the sentence is because I had come across your behaviour to minimise that.

If you do end up changing that behaviour, the example will highlight it so it should be easy to change that comment then.

Alternatively, is it the word 'adjacent' that you're objecting to?

Reply:
I think there's a crucial difference between "in the diff" as you just said versus "as different" as currently written in the example. The former denotes an issues with display representation, while the latter subtly suggests something about the semantics of how the comparison is performed.

Perhaps this is better worded as:
```go
// While the specified fields will be semantically ignored for the comparison,
// the fields may be printed in the diff when displaying entire values
// that are already determined to be different.
```